pc/storage/kdtree/kdtree.go

```diff
@@ -48,6 +48,9 @@ func (k *KDTree) newNodeArray(n0 *node) []*node {
 }
 
 func (k *KDTree) Nearest(p mat.Vec3, maxRange float32) (int, float32) {
+	if k.root == nil {
+		return -1, -1
```

Comment:
I think it's better to return `-1, maxRange*maxRange` as same as the case that the tree exists but no points are in range.